.azure-pipelines/run_hpu_tests.py

```diff
@@ -0,0 +1,142 @@
+"""This file is called from the hpu-tests.yml pipeline.
```

Comment:
@kaushikb11 could we drop this and use the standard `RunIf`?

Reply:
Yup, we could drop this and update `hpu-tests.yml` for running lightning tests.
@jerome-habana Could you create `tests/accelerators/ops_bf16_mnist.txt`, etc in the `tmp` folder within the tests itself?

Reply:
JFYI, we plan to run the tests in parallel to utilize all the cards in the dl1 instance as discussed in the first commit patch.

Reply:
moved the ops along with precision tests

Reply:
that is cool, but that means that with adding test in the codebase you need to plan it...
Still, I would highly recommend preferring readability over optimization... what s the goal having test finish in the half of the time while the machine is 95% od the time middle and we need to pay for it anyway... :/

Reply:
Also this most likely (as I read it) does not generate coverage report :(

Reply:
Would be addressed in a follow up PR